index.html

```diff
@@ -267,43 +226,53 @@ <h3 class="sidebar_list_item_head">
-                             and we'll contact you about relevant opportunities throughout the year. Previous 
-                             openings have been in the fields of cybersecurity, graphic design, web development, 
-                             user experience, and data architecture/analysis.
-                        </p>
-                        <p>
-                            Learn more about the types of work we do at our page for the
-                            <a href="http://www.consumerfinance.gov/jobs/technology-innovation-fellows/">Technology &amp; Innovation Fellowship</a> <i class="icon-external-link"></i>.
-                        </p>
-                    </div>
-                    <div>
-                        <h4 id="privacy">Privacy Act Statement</h4>
-		                <p><strong>5 U.S.C. 552(a)(e)(3)</strong></p>
-                        <p>By clicking on the 
-                            <a href="mailto:tech@cfpb.gov?subject=Interested in working in tech at CFPB">tech@cfpb.gov</a> 
-                            email address and submitting your resume, you are providing the Consumer Financial 
-                            Protection Bureau (“CFPB”) personally identifiable information for the purpose of 
-                            obtaining information on any current or future job openings, and for inclusion into 
-                            a mailing list that will be used in future communications.</p>
-                        <p>Information collected will be treated in accordance with the System of Records 
-                            Notice (“SORN”), CFPB.009 Employee Administrative Records, 78 FR 67340. This 
-                            information will not be disclosed as outlined in the Routine Uses for the SORN. 
-                            Direct identifying information will only be used to facilitate the potential 
-                            job openings, and to provide you with tailored updates from Human Resources, 
-                            and will be kept private except as required by law.</p>
-                        <p>This collection of information is authorized by Pub. L. No. 111-203, Title X, 
-                            Sections 1013 and 1022, codified at 12 U.S.C. §§ 5493 and 5512.</p>
-                        <p>Submission of your resume is voluntary. You are not required to submit or 
-                            provide any identifying information; however, not doing so may result in the CFPB 
-                            being unable to contact you about potential employment opportunities.</p>
-                    </div>
+                <div class="job-half">
+                    <p>
+                        Our mission demands smart, usable, and reliable technology.
+                        Come serve consumers by helping us design and build it.
+                    </p><p>
```

Comment:
Was the combining of the closing tag and opening tag on the same line intentional?